### ISSUE TITLE
The value_format of a Dimension is not applied to HoloMap widgets

### ISSUE THREAD

**Issue by mrksr** [ioam/holoviews#548]
A `kdim` of a `HoloMap` can have a `value_format`-function. While this function does get applied to labels, the drop-down menu rendered into a jupyter notebook seems to not be affected.

Sample Code:
```python
def data():
    return hv.Curve(np.cumsum(np.random.normal(size=(30))))

hv.HoloMap({
    'a': data(),
    'b': data(),
    'c': data()
}, kdims=[hv.Dimension("Letter", value_format=lambda x: "formatted: {}".format(x))])
```

Screenshot: ![Unformatted Drop-Down in Holomap](https://i.imgur.com/f2NCxIz.png)

**Comment by philippjfr** [comment]
Thanks for filing an issue on this. The same is true for the slider values which should in theory also support string labels, e.g. when working with timestamps that get formatted as a date. Hopefully we can get this into the next bugfix release.

**Comment by philippjfr** [comment]
This has been fixed in #562. Closing.
